src/dataMigration.js

```diff
@@ -286,6 +286,45 @@ const dataMigrations = [
       clearNumberSequences(database);
     },
   },
+  {
+    version: '8.7.0',
```

Comment:
@raviSussol : I was thinking still WP is mysterious but we are known with another case of creating blank requisition. Is there a place where it can check blank so we dont need to make everytime migrate code?